src/api/spec/features/webui/requests_spec.rb

```diff
@@ -146,7 +167,7 @@
     end
   end
 
-  context 'reviews' do
+  context 'review' do
```

Comment:
Those changes (renaming the describe labels) should get rebased with 21af38e08a2d0a5